### PR TITLE
WEBEU2 - Preprocessing II - Nabeelah Yousuph 🇳🇬

### PR DESCRIPTION
**Pull request by NabeelahY**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ Fun Bus is a travel agency looking for some help on their website.  They want a
 
 ## Task 1: Set Up The Project With Git
 
-- [ ] Create a forked copy of this project.
-- [ ] Add your project manager as collaborator on Github.
-- [ ] Clone your OWN version of the repository (Not Lambda's by mistake!).
-- [ ] Create a new branch: git checkout -b `<firstName-lastName>`.
-- [ ] Implement the project on your newly created `<firstName-lastName>` branch, committing changes regularly.
-- [ ] Push commits: git push origin `<firstName-lastName>`.
+- [✓] Create a forked copy of this project.
```
Comment (maximesalomon) [inline README.md:7]: Great job using the README to check your progress!

File: index.html [hunk 2]
```diff
@@ -1,9 +1,10 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
-
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (maximesalomon) [inline index.html:7]: 👍

File: index.html [hunk 3]
```diff
@@ -15,53 +16,81 @@
...
+      <a href='index.html'>Home</a>
+      <a href='#'>About Us</a>
+      <a href='#'>Blog</a>
+      <a href='#'>Contact</a>
+    </nav>
+  </header>
 
-  
-     
   <div class="container home">
```
Comment (maximesalomon) [inline index.html:33]: no `<main>`?

File: less/home-page.less [hunk 4]
```diff
@@ -1,21 +1,71 @@
+/* HEADER */
+header {
+  .pad-margin(1em);
+  .flexbox-style(flex, space-between);
+  @media @mobile {
+    .flexbox-style(flex, center);
```
Comment (maximesalomon) [inline less/home-page.less:6]: Niice

File: less/mixins.less [hunk 5]
```diff
@@ -1 +1,15 @@
...
+    display: @display;
+    justify-content: @direction;
+}
+
+.pad-margin (@size) {
+    margin: @size;
+    padding: @size; 
+}
+
+.btn-style (@width, @radius) {
```
Comment (maximesalomon) [inline less/mixins.less:12]: 👍

File: less/variables.less [hunk 6]
```diff
@@ -11,4 +11,8 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+// Breakpoints
+@mobile: ~"(max-width: 500px)";
```
Comment (maximesalomon) [inline less/variables.less:17]: 👍

**Review by maximesalomon — COMMENTED** [review]
Great job with this project Nabeelah 🔥

Your code looks clean and you successfully reached MVP

MPV = ✅
* Imported LESS Files ✅
* Implemented the navigation, header, and button ✅
* Desktop ✅
* Used a parametric mixin for your buttons ✅
* Used escaping to create a @mobile variable ✅
* Mobile ✅

Try to use the git commit guidelines that we talked about - https://chris.beams.io/posts/git-commit/

You got this 💪

**Comment by NabeelahY** [comment]
Thank you!

**Pull request merged**